pkg/util/oc/pod_commands.go

```diff
@@ -83,3 +90,7 @@ func WaitAllPodsReady(t test.TestHelper, ns string) {
 	t.T().Helper()
 	shell.Executef(t, `oc -n %s wait --for condition=Ready --all pods --timeout 180s`, ns)
 }
+func WaitCondition(t test.TestHelper, ns string, rs string, rsname string, condition string, timeout string) {
```

Comment:
rename:
- `rs` to `kind`
- `rsname` to `name`

`timeout` probably doesn't have to be a parameter, at least for now. We can hardcode it to a sensible value (60s, 300s?)

Reply:
Done, set the timeout "hardcoded" is the "middle" to `"180s"`